mtags/src/main/scala-2/scala/meta/internal/metals/TokenEditDistance.scala

```diff
@@ -289,6 +289,8 @@ object TokenEditDistance {
     if (!isScala) {
       // Ignore non-scala Files.
       unchanged
+    } else if (originalInput.value.isEmpty() || revisedInput.value.isEmpty()) {
```

Comment:
This could break otherwise. This situation means that we most likely do not know how the file looked when the semanticdb was compiled. This will result in anything depending on edit distance not to work, but will not create a risk for incorrect results.

Reply:
Also if one of them is empty we cannot match any tokens.